sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -395,11 +399,17 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       offeringExistsQuery.andWhere("programYear.active = true");
     }
     offeringExistsQuery.select("1");
-    return this.repo
+    const programsQuery = this.repo
       .createQueryBuilder("programs")
       .where("programs.programStatus = :programStatus", {
         programStatus: ProgramStatus.Approved,
-      })
+      });
+    if (!isFulltimeAllowed) {
```

Comment:
IMO environment based logic must not be used inside the services.

Service must receive an optional parameter which should lead to where condition.

Reply:
@dheepak-aot I see the point but this is supposed to be a temporary solution that would be easier to be removed in this way.
I do not have a strong feeling in one way or another but I also did not get what it would not be recommended to use the config service inside a service.

Reply:
I have changed it (for now) considering the explanation from @dheepak-aot that if it is passed as a parameter, then the service becomes more configurable allowing the same parameter to be passed from a different service with a different value.